benchmarking/benchmarks/benchmarks.py

```diff
@@ -64,15 +64,15 @@ def _collectOneBenchmark(self, source, meta, benchmarks, info):
 
         self._updateFiles(one_benchmark, source)
 
-        self._updateTests(one_benchmark, source)
-
         if meta:
```

Comment:
can you please add a comment here saying the following modifications should not appear in the updated json. Thanks.